src/main/kotlin/no/skatteetaten/aurora/boober/feature/FluentbitSidecarFeature.kt

```diff
@@ -238,8 +238,8 @@ class FluentbitSidecarFeature(
             resources {
                 limits = mapOf(
                     // TODO? Add as config parameter
-                    "memory" to Quantity("64Mi"),
-                    "cpu" to Quantity("100m")
+                    "memory" to Quantity("128Mi"),
```

Comment:
Burde dette også være config lik det jeg sier under her?

Reply:
Spm er om det skal konfigureres i boober eller for hver app? Kanskje vi bare kan la det være slik det er nå inntil videre?

Reply:
Egen sak: https://jira.sits.no/browse/AOS-5306